### ISSUE TITLE
opdk: mesg: ttyname failed: Inappropriate ioctl for device

### ISSUE THREAD

**Issue by MateusAmin** [stackinabox/stackinabox.io#1]
The up seems to have failed and I get this error message in the output. Thanks in advance!

opdk: mesg: ttyname failed: Inappropriate ioctl for device

```
opdk: Checking for Docker Compose installation...
opdk: Symlinking Docker Compose 1.8.0
opdk: Running docker-compose up...
```

==> opdk: mesg: ttyname failed: Inappropriate ioctl for device
==> opdk: Creating ucddb
==> opdk: Creating blueprintdb
==> opdk: Creating heatengine
==> opdk: Creating ucd
==> opdk: Creating blueprintdesigner
==> opdk: Creating agent
==> opdk: Creating agent-relay

**Comment by tpouyer** [comment]
The following will occur anytime the Vagrantfile executes a script as part of provisioning when using a Ubuntu 16.04 based vagrant box.  

> mess: ttyname failed: Inappropriate ioctl for device

You can safely ignore these errors.  The other messages are coming from the vagrant-docker-compose plugin.  It's logging the fact that it has started the process of "up'ing" the UrbanCode docker containers.

**Comment by MateusAmin** [comment]
Cool, thank you! I thought something might be wrong because this link (http://designer.stackinabox.io:9080/) wouldn't work.

**Comment by MateusAmin** [comment]
Web terminal does work though.

**Comment by tpouyer** [comment]
Seems that the links to 'Blueprint Designer' in the README.md file need to be updated.

**Comment by tpouyer** [comment]
Finally found a fix for this one.  Seems that the ubuntu 16.04 release changed slightly the shell that is exposed over ssh.  You can remove these errors by adding the following to the Vagrantfile:

`config.ssh.shell = "bash -c 'BASH_ENV=/etc/profile exec bash'"`

I'll include this in the next release.  Until then you can ignore those errors.

**Comment by tpouyer** [comment]
fixed in latest release 0.9.7
